loadtest/control/simulcontroller/actions.go

```diff
@@ -117,6 +129,11 @@ func (c *SimulController) login(u user.User) control.UserActionResponse {
 			c.status <- c.newErrorStatus(err)
 		}
 
+		errId := resp.Err.(*control.UserError).Err.(*model.AppError).Id
+		if errId == "api.user.login.invalid_credentials_email_username" {
+			return resp
+		}
```

Comment:
Would this lift a bit? 😅 
```suggestion
		if appErr, ok := resp.Err.(*control.UserError).Err.(*model.AppError); ok && appErr.Id == "api.user.login.invalid_credentials_email_username" {
			return resp
		}
```

Reply:
That makes it even longer :D